services/blockchain-connector/shared/sdk/events.js

```diff
@@ -14,10 +14,13 @@
  *
  */
 const util = require('util');
-const { Signals, Logger } = require('lisk-service-framework');
+const { Signals, Logger, Exceptions: { TimeoutException } } = require('lisk-service-framework');
 
+const BluebirdPromise = require('bluebird');
+const { getApiClient, invokeEndpoint, timeoutMessage } = require('./client');
 const { getRegisteredEvents } = require('./endpoints');
-const { getApiClient } = require('./client');
+const { decodeEventPayload } = require('./decoder');
+const { getEventID, getEventSchemaFromName } = require('../utils/events');
```

Comment:
Please consider renaming the methods based on their functionality.
```suggestion
const { computeEventID, getEventSchemaByName } = require('../utils/events');
```

Reply:
Renamed to `calculateEventID`